oauth2/introspector_test.go

```diff
@@ -62,36 +63,36 @@ var localWarden = &warden.LocalWarden{
 
 func init() {
 	introspectors["local"] = &oauth2.LocalIntrospector{
-		OAuth2: localWarden.OAuth2,
-		Issuer: "tests",
+		OAuth2:              localWarden.OAuth2,
+		Issuer:              "tests",
 		AccessTokenLifespan: time.Hour,
 	}
 
 	r := httprouter.New()
 	serv := &oauth2.Handler{
-		Firewall: localWarden,
-		H:      &herodot.JSON{},
+		Firewall:     localWarden,
+		H:            &herodot.JSON{},
 		Introspector: introspectors["local"],
 	}
 	serv.SetRoutes(r)
 	ts = httptest.NewServer(r)
 
 	ar := fosite.NewAccessRequest(oauth2.NewSession("alice"))
-	ar.GrantedScopes = fosite.Arguments{"core"}
+	ar.GrantedScopes = fosite.Arguments{0: "core"}
 	ar.RequestedAt = now
 	ar.Client = &fosite.DefaultClient{ID: "siri"}
 	ar.Session.(*oauth2.Session).Extra = map[string]interface{}{"foo": "bar"}
 	fositeStore.CreateAccessTokenSession(nil, tokens[0][0], ar)
 
 	ar2 := fosite.NewAccessRequest(oauth2.NewSession("siri"))
-	ar2.GrantedScopes = fosite.Arguments{"core"}
+	ar2.GrantedScopes = fosite.Arguments{0: "core"}
 	ar2.RequestedAt = now
 	ar2.Session.(*oauth2.Session).Extra = map[string]interface{}{"foo": "bar"}
 	ar2.Client = &fosite.DefaultClient{ID: "siri"}
 	fositeStore.CreateAccessTokenSession(nil, tokens[1][0], ar2)
 
 	ar3 := fosite.NewAccessRequest(oauth2.NewSession("siri"))
-	ar3.GrantedScopes = fosite.Arguments{"core"}
+	ar3.GrantedScopes = fosite.Arguments{0: "core"}
```

Comment:
Yeah ;) I think i'll remove the 0's. Will just confuse lesser bewildered gophers.